src/main/java/com/google/sps/servlets/StartGame.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.sps.daos.RoomDao;
+
+@WebServlet("/startGame")
+public class StartGame extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+
+    // Activate room
+    RoomDao dao = (RoomDao) this.getServletContext().getAttribute("roomDao");
+    dao.startGame();
```

Comment:
I think Brett is right here. If we want to do something like `thing.startGame()`, that thing should probably not be the RoomDao. Maybe RoomHelper/GameInstanceHelper or something?

Reply:
Yeah it is better I will try to change everything tomorrow :)

Reply:
I'll do this after this PR merge :)